skimage/measure/fit.py

```diff
@@ -687,12 +688,29 @@ def _dynamic_max_trials(n_inliers, n_samples, min_samples, probability):
         return 0
 
     return int(np.ceil(nom / denom))
+    
+    
+def check_random_state(seed):
```

Comment:
This is a useful function for other parts of skimage as well. Would you mind moving this to e.g. skimage._shared.util?